autotest/gcore/basic_test.py

```diff
@@ -40,11 +40,13 @@
 # Nothing exciting here. Just trying to open non existing files,
 # or empty names, or files that are not valid datasets...
 
+non_existing_error_msg = 'No such file or directory'
```

Comment:
Actually, the way you have the branches set up, I don't believe that will work. I'll try to find another way to test on Windows.